samples/ar-view-node/src/main/java/io/github/sceneview/sample/arviewnode/nodes/ViewNodeHelper.kt

```diff
@@ -0,0 +1,86 @@
+package io.github.sceneview.sample.arviewnode.nodes
+
+import android.content.Context
+import android.view.View
+import android.widget.TextView
+import com.google.ar.core.Anchor
+import com.google.ar.sceneform.rendering.ViewAttachmentManager
+import com.google.ar.sceneform.rendering.ViewRenderable
+import io.github.sceneview.ar.ARSceneView
+import io.github.sceneview.ar.node.AnchorNode
+import io.github.sceneview.math.Scale
+import io.github.sceneview.node.ViewNode
+import io.github.sceneview.sample.arviewnode.R
+import io.github.sceneview.sample.arviewnode.nodes.events.ViewNodeEvent
+import java.util.Locale
+import java.util.concurrent.CompletableFuture
+
+class ViewNodeHelper(
+    private val context: Context,
+    private val sceneView: ARSceneView,
+    val onEvent: (ViewNodeEvent) -> Unit
```

Comment:
Should we give a more descriptive name to the callback?

Reply:
On this side not. It would make sense to give a more descriptive name to the actual call function in MainActivity.

Reply:
I renamed the callback function